src/components/CityWeatherShortcut/CityWeatherShortcut.tsx

```diff
@@ -27,7 +26,7 @@ const CityWeatherShortcut = ({ cityName }: Props) => {
           <Value>{roundedTemperature}</Value>
           <TemperatureSign>&deg;C</TemperatureSign>
         </Temperature>
-        <City>{data.location.name}</City>
+        <City>{cityName}</City>
```

Comment:
**Question** Why did you change it. Is there any reason to do so?

Reply:
I'll change it, this is our leftover from application testing